### PR TITLE
[4.0-7.9] [#2483] Fix current selected security section is not highlighted in the menu

### PR DESCRIPTION
**Pull request by frankeros**
Hello team, this PR resolves:

- Current section is not highlighted in the main Menu

closes [#2483](https://github.com/wazuh/wazuh-kibana-app/issues/2483)

### CODE REVIEW

File: public/redux/actions/securityActions.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/*
+ * Wazuh app - React component for registering agents.
```
Comment (jsanchez91) [inline public/redux/actions/securityActions.js:2]: Remember to change the description of the copyright block

File: public/redux/reducers/securityReducers.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*
+ * Wazuh app - React component for registering agents.
```
Comment (jsanchez91) [inline public/redux/reducers/securityReducers.js:2]: The same problem

**Review by jsanchez91 — CHANGES_REQUESTED** [review]

**Comment by jsanchez91** [comment]
@frankeros,

After testing the code, I found this bug:
> When you leave the security section, the item "Users" remains selected
selected_security_tab

**Comment by frankeros** [comment]
> @frankeros,
> 
> After testing the code, I found this bug:
> 
> > When you leave the security section, the item "Users" remains selected
> > selected_security_tab

@jsanchez91 this bug was fixed in this [commit](https://github.com/wazuh/wazuh-kibana-app/pull/2567/commits/9755f9468168c1b7c4010e8d5407f2fa69ea565b)

**Review by jsanchez91 — APPROVED** [review]
LGTM! :tada:

**Pull request merged**